src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.html

```diff
@@ -30,8 +32,10 @@
             <span class="message">{{ line.message }}</span>
           </p>
 
-          <span *ngIf="contentData.audit_log.length === 0"
-                i18n>No entries found</span>
+          <span *ngIf="audit_log.length === 0"
+                i18n>
+            No log entries found. Please try to select different filter options.
+          </span>
```

Comment:
```suggestion
                i18n>No log entries found. Please try to select different filter options.</span>
```

Reply:
Hmm I also tested this locally but it did not fail :)
I going to tackle this next week I'm currently busy with the branding stuff.